__tests__/app.test.js

```diff
@@ -1,3 +1,5 @@
+//  Begining of default test code
```

Comment:
This test suite is likely to fail because jest uses a test environment called `jsdom` but you have to set it. Usually configurations for jest are set in a `jest.config.js` file.
You can read up on it here=> https://jestjs.io/docs/en/configuration#testenvironment-string, https://jestjs.io/docs/en/configuration

Reply:
think I fixed this too